File: laramie-importacion-webapi/src/domain/entities/configuracion.ts
export default class Configuracion {

    id: number;
	nombre: string;
	valor: string;

	constructor(
        id: number = 0,
		nombre: string = "",
		valor: string = ""
	)
	{
        this.id = id;
		this.nombre = nombre;
		this.valor = valor;
	}

	setFromObject = (row) =>
	{
        this.id = row.id ?? 0;
		this.nombre = row.nombre ?? "";
		this.valor = row.valor ?? "";
	}

}

File: modules/tesoreria/models/configuracion.py
from sqlalchemy import Column, BigInteger, String

import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", ".."))
from shared.database import Base


class Configuracion(Base):
    __tablename__ = "tesoreria_configuraciones"

    id = Column(BigInteger, primary_key=True)
    nombre = Column(String(250), nullable=False)
    valor = Column(String(1000), nullable=False)
